index.js

```diff
@@ -0,0 +1,28 @@
+const calculate = (calculatorInputs) => {
```

Comment:
This could benefit from some helper functions. Functions should do one thing and your function is currently doing at least 3.